README.md

```diff
@@ -282,6 +282,86 @@ When running in silent mode, o-typography does not load web fonts, products shou
 
 When running in non-silent mode, o-typography loads all web fonts which are used.
 
+### Progressively loading web fonts
```

Comment:
Can you add that this feature isn't available if you're using the Build Service?
I'm going to open a seperate issue to discuss whether it *should* be, but that shouldn't hold up this PR being merged.